vyper/exceptions.py

```diff
@@ -116,6 +120,7 @@ def __init__(self, message, source_code, lineno, col_offset):
         item.lineno = lineno
         item.col_offset = col_offset
         item.full_source_code = source_code
+        print("got this far")
```

Comment:
```suggestion
```

Reply:
This is the real reason for code reviews.